Source/Charts/Charts/BarLineChartViewBase.swift

```diff
@@ -555,6 +568,29 @@ open class BarLineChartViewBase: ChartViewBase, BarLineScatterCandleBubbleChartD
         }
     }
     
+    @objc private func longPressGestureRecognized(_ recognizer: NSUITapGestureRecognizer)
+    {
+        if data === nil
+        {
+            return
+        }
+        
+        if recognizer.state == NSUIGestureRecognizerState.began ||
+            recognizer.state == NSUIGestureRecognizerState.changed
+        {
+            if !isHighLightPerTapEnabled { return }
+            
+            let h = getHighlightByTouchPoint(recognizer.location(in: self))
```

Comment:
@robert-doan whereever possible we prefer to use more descriptive variable names instead of just `h`